packages/flutter/test/cupertino/nav_bar_transition_test.dart

```diff
@@ -1020,6 +1020,76 @@ void main() {
     );
   });
 
+  testWidgets('Bottom CupertinoSliverNavigationBar.bottom fades and slides in from the left', (
+    WidgetTester tester,
+  ) async {
+    await startTransitionBetween(
+      tester,
+      from: const CupertinoSliverNavigationBar(
+        bottom: PreferredSize(preferredSize: Size.fromHeight(30.0), child: Placeholder()),
+      ),
+      fromTitle: 'Page 1',
+    );
+
+    await tester.pump(const Duration(milliseconds: 50));
+
+    // There's 2, one from the bottom large title fading out and one from the
+    // bottom back label fading in.
+    expect(flying(tester, find.text('Page 1')), findsNWidgets(2));
+    expect(flying(tester, find.byType(Placeholder)), findsOneWidget);
+
+    checkOpacity(tester, flying(tester, find.byType(Placeholder)), 0.946);
+
+    expect(
+      tester.getTopLeft(flying(tester, find.byType(Placeholder))).dx,
+      moreOrLessEquals(-20.58, epsilon: 0.01),
+    );
+
+    await tester.pump(const Duration(milliseconds: 200));
+
+    checkOpacity(tester, flying(tester, find.byType(Placeholder)), 0.001);
```

Comment:
The transition is still playing at this point, right? This is about halfway through? Could you add a comment explaining where in the transition this is?